### DISCUSSION TITLE
Shipping Method Error in v6

### DISCUSSION THREAD

**Discussion by bellevue7k-user** [duncanmcclean/simple-commerce#1020] [Help]
I keep getting this error in the new version. I created my own method, but I get the same error.
Even when I use this instead of my own method, I get an error.
  
// simple-commerce.php 
```
   'shipping' => [
       'default_method' => 'free_shipping',
      
       'methods' => [
           \DuncanMcClean\SimpleCommerce\Shipping\FreeShipping::class => [],
       ],
   ],
```
//Error

Class "free_shipping" not found {"exception":"[object] (Error(code: 0): Class \"free_shipping\" not found at /Users/......../Sites/......./vendor/duncanmcclean/simple-commerce/src/Orders/Calculator/ShippingCalculator.php:23)

**Comment by duncanmcclean** [comment]
It's a bug - see #1019.
